### PR TITLE
Switched LC fitting to bokeh

### PR DESCRIPTION
**Pull request by hover2pi**
Switching to Bokeh and getting plotting in ship shape.

### CODE REVIEW

**Comment by hover2pi** [comment]
I also added a `results` attribute to the `LightCurve` object to store and plot best fits dynamically.

**Comment by hover2pi** [comment]
Ready to go!

**Comment by hover2pi** [comment]
Wait @bourque ! Not ready to go.

**Comment by hover2pi** [comment]
@bourque Ok NOW it's good to go.

File: exoctk/lightcurve_fitting/fitters.py [hunk 1]
```diff
@@ -83,7 +91,7 @@ def lmfitter(time, data, model, unc=None, method='leastsq', verbose=True):
 
     # Make a new model instance
     best_model = copy.copy(model)
-    best_model.name = 'Best Fit'
     best_model.parameters = params
+    best_model.name = ', '.join(['{}:{}'.format(k,round(v[0], 2)) for k,v in params.dict.items()])
```
Comment (bourque) [inline exoctk/lightcurve_fitting/fitters.py]: @hover2pi Technically according to PEP8 this should be with some extra whitespace:

```python
best_model.name = ', '.join(['{}:{}'.format(k, round(v[0], 2)) for k, v in params.dict.items()])
```

File: exoctk/tests/test_lightcurve_fitting.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
+#! /usr/bin/env python
```
Comment (bourque) [inline exoctk/tests/test_lightcurve_fitting.py]: @hover2pi You may want to add this module to the `ExoCTK.tests.rst` API docs as well.

File: exoctk/lightcurve_fitting/fitters.py (hunk 1 above)
Comment (hover2pi) [reply exoctk/lightcurve_fitting/fitters.py]: You caught me. I didn't flake8 it!

File: .pytest_cache/v/cache/nodeids [hunk 3]
```diff
@@ -1 +1,9 @@
-[]
\ No newline at end of file
+[
```
[the remark this answers is not in the document]
Comment (hover2pi) [reply .pytest_cache/v/cache/nodeids]: Hmmm, it's not on the `.gitignore` I guess.

File: exoctk/lightcurve_fitting/fitters.py [hunk 4]
```diff
@@ -21,6 +21,12 @@ def lmfitter(time, data, model, unc=None, method='leastsq', verbose=True):
         The model to fit
     unc: np.ndarray (optional)
```
[the remark this answers is not in the document]
Comment (hover2pi) [reply exoctk/lightcurve_fitting/fitters.py]: Laziness.

**Pull request merged**